pkg/api/kptfile/v1alpha2/validation.go

```diff
@@ -27,8 +28,10 @@ const (
 )
 
 func (kf *KptFile) Validate() error {
+	const op errors.Op = "kptfile.validate"
```

Comment:
Not sure how should we construct this `op` string. Should it be `[pkgName].[structName].[funcName]` or any string that can represent current operation?

Reply:
Checked with Morten and he said the convention for now is <package name>.<function name>. I will update the PR.

Reply:
updated it to include the package name

Reply:
We may need to standardize the format of this string. And it will be good to document how to use this `errors` package. @droot @mortent

Reply:
I agree, this should be addressed as part of #1783 and #1557